app/src/androidTest/java/com/github/houseorganizer/houseorganizer/MainScreenActivityTest.java

```diff
@@ -225,12 +255,12 @@ public void signOutButtonIsClickable(){
     }
 
     @Test
-    public void signOutButtonFiresRightIntent(){
+    public void zSignOutButtonFiresRightIntent(){
```

Comment:
This seems a bit weird having to start test names with a z in order to have it run after the other ones. If we go this way, we should at least normalize formatting i.e test01<name>; test02<name> etc.

Reply:
Maybe there is a way to force the order of tests. We could investigate this in a future sprint

Reply:
Yes good idea. An alternative would be to use @Order(int) before each test but that would mean adding it to all the existing tests